app/views/markdown/help.md

```diff
@@ -0,0 +1 @@
+help page
```

Comment:
Did you want anything here?

Reply:
Yes, but I'd like to do it in a future PR.

Reply:
@srerickson Will that PR be before release? If so, no problem. However, if it's going to be after 4.0, we should remove the page and the link from top bar.